cli/tauri.js/test/jest/__tests__/build.spec.js

```diff
@@ -0,0 +1,77 @@
+const path = require('path')
+const fixtureSetup = require('../fixtures/app-test-setup')
+const appDir = fixtureSetup.appDir
+const distDir = fixtureSetup.distDir
+
+const spawn = require('../../../src/helpers/spawn').spawn
+
+function runBuildTest(tauriConfig) {
+  fixtureSetup.initJest()
+  const build = require('../../../src/api/build')
+  return new Promise(async (resolve, reject) => {
+    try {
+      let appPid
+      const server = fixtureSetup.startServer(() => appPid, resolve)
+      const result = build(tauriConfig)
+      await result.promise
+
+      const artifactFolder = tauriConfig.ctx.debug ? 'debug' : 'release'
+      const artifactPath = path.resolve(appDir, `src-tauri/target/${artifactFolder}/app`)
+
+      appPid = spawn(
+        process.platform === 'win32' ? `${artifactPath}.exe` : artifactPath.replace(`${artifactFolder}/app`, `${artifactFolder}/./app`),
```

Comment:
Why is this necessary on windows? Maybe add a comment so it doesn't get removed on accident.